modules/2_classification/README.md

```diff
@@ -45,4 +45,6 @@ deep learning network. To complete this module, you should implement:
 
 * Put `DenseNet_121.xml` and `DenseNet_121.bin` to `openvino_practice/data` folder.
 
-* Use `git` to add model files (`.xml` and `.bin`) into commit
+* Add code in `src/classifier.cpp` and `test/test_classifier.cpp`.
```

Comment:
I'd like to recommend to change current approach a little bit. Doing practice in the test was bad idea. Let's modify a method `void classify(const cv::Mat& image, std::vector<float>& probabilities);` so in the following way:
```cpp
void classify(const cv::Mat& image, int k, std::vector<float>& probabilities, std::vector<int>& indices);
```
So test will just call it but student have to implement the code in the `src/classifier.cpp`.

Note that you don't have to modify https://github.com/dkurt/openvino_practice/pull/26 - it's accepted. Let's make the changes here.